arch/arm/core/aarch64/macro_priv.inc

```diff
@@ -19,7 +19,7 @@
  * @return N/A
  */
 
-.macro z_arm64_enter_exc xreg0, xreg1, xreg2
+.macro z_arm64_save_context xreg0, xreg1, xreg2
```

Comment:
Well, also `z_arm64_{enter,exit}_exc` is not an entirely proper name either since they are being used when entering and exiting the exception but the name is not descriptive of what's going on in the macros

Reply:
I find `z_arm64_{enter,exit}_exc` to be better.. as these (`x0-18, LR, SPSR, ELR`) seem to be the registers which need to be saved on exception/interrupt.. I will leave it to you.

Reply:
No prob. I have removed the renaming commit. Thanks.